plasmapy/utils/decorators/converter.py

```diff
@@ -0,0 +1,28 @@
+"""
+Decorator to convert units of functions in /physics methods
+"""
+__all__ = ["angular_freq_to_hz"]
+
+import functools
+import inspect
+from astropy import units as u
+from plasmapy.utils.decorators import preserve_signature
+
+
+def angular_freq_to_hz(fn):
+
+    fn.__doc__ += """
+    Parameters
```

Comment:
Oh wow, this works! Awesome!

https://2687-46810954-gh.circle-artifacts.com/0/root/project/docs/_build/html/api/plasmapy.physics.parameters.gyrofrequency.html?highlight=gyrofrequency

Maybe, so as to not duplicate the Parameters heading:

```suggestion
    Additional parameters
```

Of course it would be amazing to put it along with `Parameters`, but let's skip the parsing.

Reply:
Yeah, that's the same result I got when testing this locally.  The `to_hz` parameter definition is inject at the very bottom of the documentation.   This still happens identically when I used the ["Other Parameters" heading suggested by the numpydoc documentation](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard).  The one downside, the signature/function heading (the blue heading) does not get updated with `to_hz=False`.

I've been searching though the sphinx documentation to see how this heading is generated but its still not clear to me.  If this is okay with you, then we could make this an issue for future improvement. ¯\\_(ツ)_/¯

Reply:
@diego7319 and @StanczakDominik, I think I figured out how to update the signature as well...

``` python
def angular_freq_to_hz(fn):
    # raise exception if fn uses the 'to_hz' kwarg
    sig = inspect.signature(fn)
    if 'to_hz' in sig.parameters:
        raise ValueError

    # make new signature for fn
    new_params = sig.parameters.copy()
    new_params['to_hz'] = inspect.Parameter('to_hz', inspect.Parameter.POSITIONAL_OR_KEYWORD, default=False)
    new_sig = inspect.Signature(parameters=new_params.values(), return_annotation=sig.return_annotation)
    fn.__signature__ = new_sig

    # add 'to_hz' to fn docstring
    fn.__doc__ = """
    Other Parameters
    ----------------
    to_hz: bool
        Set `True` to to convert function output from angular frequency to Hz
    """

    def wrapper(*args, to_hz=False, **kwargs):
        ...
```

This accomplished everything we wanted, except placing `'to_hz'` into the original parameters section, on my local documentation build.